documentation/build.md

```diff
@@ -33,75 +51,92 @@ I... I would never challenge you to do stupid things...
+However the mapping was not chosen randomly and should only be changed if you know what you are doing.
+Since there now are 2 voltage regulators, the power draw should be split as even as possible between the regulators.
+The mapping was chosen based on this criteria.
 
 | Channel   | Mapped to        |
 | --------- | ---------------- |
 | Channel 1 | Dash             |
-| Channel 2 | Centre console   |
-| Channel 3 | Front left door  |
-| Channel 4 | Front right door |
-| Channel 5 | Rear left door   |
-| Channel 6 | Rear right door  |
+| Channel 2 | Front left door  |
+| Channel 3 | Rear left door   |
+| Channel 4 | Footwell left    |
+| Channel 5 | Center console   |
+| Channel 6 | Front right door |
+| Channel 7 | Rear right door  |
+| Channel 8 | Footwell right   |
 
 At the bottom of the PCB there are 2 connectors, a micro USB port and a micro SD card slot.
 
 The 4 pin XH connector is for powering the board.
-When you use the onboard regulator, this can be up to 45V (absolute maximum rating).
-When an external regulator is used, the jumper J1 has to be bridged and the input voltage must be stable 5V.
-In that case there should be no regulator on the TesLight board.
-Never bridge J1 when the onboard regulator is used.
-This will short the regulator and could destroy your hardware.
+When you use the integrated regulator, the input voltage can be up to 45V (absolute maximum rating).
+When an external regulator is used, the jumpers `J1` and `J2` must be bridged and the input voltage must be 5V!
+In that case there should be no regulator components on the TesLight board.
+
+> !!! Never bridge `J1` when the onboard regulator is used !!!<br>
+> This will connect the regulated 5V output to the input and will most likely destroy your hardware completely.
 
 The 3 pin XH connector can be used as a sensor to measure the voltage of your car's ambient light and supply TesLight an on/off or brightness signal.
-The pin can be used with analog voltages and also PWM signals.
-It is recommended to not measure voltages above 20V with this controller.
-There is an overvoltage protection in place and exceeding this limit for a short period is possible but not recommended.
+The pin can be used with analog voltages and also PWM signals due to the low-pass filter.
+It is recommended to not measure voltages above 19V with this controller.
+There is an overvoltage protection in place and exceeding this limit for a short period should be possible but not recommended.
+Instead the resistor `R10` of the voltage devider can be adjusted accordingly.
+
 Keep in mind that according to the picture diagram above, the right pin is connected to ground.
-Never connect it to the positive wire of your car lights.
+Never connect it to the positive wire of your car lights when using them as signal to turn on and off.
 If you are sure that there is a good ground connection, which is usually the case, there is no need to use it.
```

Comment:
Meinst du hier den Groundpin von der originalen Fußraumbeleuchtung? Würde ich noch etwas klarer rüberbringen :) Ich werde den Groundpin wahrscheinlich trotzdem anschließen, einfach nur um mein inneres Ich zu befriedigen :D

Reply:
What I mean here is that the ground pin of the sensor connector should never be connected to the positive of the footwell lights. Just a warning to pay attention. Since its the same ground connection as for the power input, it would short the lights positive wire to ground. Probably not a good idea 😋 . 

Connecting both is perfectly fine btw.